index.js

```diff
@@ -74,6 +75,8 @@ function translate(text, opts) {
             body[0].forEach(function (obj) {
                 if (obj[0]) {
                     result.text += obj[0];
+                } else if (obj[2]) {
```

Comment:
Shouldn't it be here just `if` with out `else`? It seems to me that text and punctuation are independent..
E.g.:
```js
if (obj[0]) {
  result.text += obj[0];
}

if (obj[2]) {
  result.pronunciation += obj[2];
}
```

Reply:
That makes sense. I could do that. I'll resolve this when done.

Reply:
Makes sense here.